dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/publish/servers/jakartasec-3.0_fat.config.op/configs/server_orig.xml

```diff
@@ -17,6 +17,7 @@
 	<include location="${shared.config.dir}/OPMisc.xml" />
 	<include location="${shared.config.dir}/oauthRoles_1.xml" />
 	<include location="${shared.config.dir}/oidcProvider.xml" />
+	<include location="${shared.config.dir}/oidcProvider_useNonce.xml" />
```

Comment:
In the future replace oidcProvider with the test specific provider in a new server config file and use that your tests. (in this case replace oidcProvider.xml with oidcProvider_useNone.xml )